src/ontology/reports/omim_mapped_deprecated_terms.robot.template.tsv

```diff
@@ -0,0 +1,92 @@
+mondo_id	omim_id	exclusion_reason
+ID	ID	AI MONDOREL:has_exclusion_reason
+MONDO:0000911	OMIM:613740	MONDO:equivalentObsolete
```

Comment:
@matentzn I think that when you asked me to add `MONDO:equivalentObsolete` in this column, it is an `exclusion_reason`. But FYI, it doesn't look like we have that exact entry in [the exclusion list](https://mondo.readthedocs.io/en/latest/editors-guide/exclusion-reasons/) yet.

Reply:
No no, its not an exclusion reason, made a fix. Will discuss details later.

Reply:
Oh nooo, I just overwrote your commit; not sure what you changed. I'm so sorry.

Reply:
Oh, I see your [suggestion](https://github.com/monarch-initiative/mondo-ingest/pull/108#discussion_r1035823464), `A oboInOwl:source`. I'll add that to the 2nd row, and change the header to `source` for now.

Reply:
Nice, thanks